broker-core/src/main/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessor.java

```diff
@@ -70,6 +70,9 @@ public ExporterStreamProcessor(
     this.containers = new ArrayList<>(descriptors.size());
     for (final ExporterDescriptor descriptor : descriptors) {
       this.containers.add(new ExporterContainer(descriptor));
+      if (state.getPosition(descriptor.getId()) == ExporterStreamProcessorState.VALUE_NOT_FOUND) {
+        state.setPosition(descriptor.getId(), -1L);
+      }
```

Comment:
We should move this function to `onRecovered()`. We already update the state there :)

Reply:
Please add a new test case to `ExporterStreamProcessorTest` to verify that we don't recover from a higher position if one exporter never updates its position. It shouldn't be hard to write this as unit test ;)

Also, we should add a test case in `ExporterStateTest` for the new method `getLowestPosition()`.